internal/proto/jobmanager_service.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/devnulled/runjob/internal/proto";
+
+package jobmanager;
+
+//
+// JobManager
+//
+// The JobManager is the service on the server responsible for starting/managing/stopping running jobs
+//
+service JobManager {
+    rpc Start(JobStartRequest) returns (JobStartResponse);
+    rpc Stop(JobStopRequest) returns (JobStopResponse);
+    rpc Status(JobStatusRequest) returns (JobStatusResponse);
+    rpc Stream(JobStreamRequest) returns (stream JobStreamResponse);
+    // Ping is a simple request just to test connectivity and certifcate validation
+    rpc Ping(PingRequest) returns (PingResponse);
+}
+
+
+message JobStopRequest {
+    // The Job ID to stop
+    string job_id = 1;
+}
+
+message JobStopResponse {}
+
+message JobStatusRequest {
+    string job_id = 1;
+}
+
+message JobStatusResponse {
+    JobStatus status = 1;
+}
+
+message JobStreamRequest {
+    string job_id = 1;
+}
+
+message JobStreamResponse {
+    bytes response = 1;
```

Comment:
nit: The name response is quite vague for a gRPC response message field in the context of
 Request/Response messages:
 
```suggestion
    bytes output = 1;
```